tests/lca/imagebasedinstall/mgmt/deploy/tests/e2e-deploy-test.go

```diff
@@ -79,11 +87,34 @@ var _ = Describe(
 					Skip("Cluster must use static networking")
 				}
 
+				if MGMTConfig.SiteConfig {
+					Skip("Cluster deployed through siteconfig operator")
+				}
+
 				tsparams.ReporterNamespacesToDump[MGMTConfig.Cluster.Info.ClusterName] = "spoke namespace"
 
 				createIBIOResouces(ipv4AddrFamily)
 			})
 
+		It("through siteconfig operator is successful in an IPv6 proxy-enabled environment with DHCP networking",
+			reportxml.ID("76642"), func() {
+				if MGMTConfig.StaticNetworking {
+					Skip("Cluster must use DHCP networking")
+				}
+
+				if !MGMTConfig.SiteConfig {
+					Skip("Cluster must be deployed through siteconfig operator")
```

Comment:
```suggestion
					Skip("Cluster was not opted to be deployed through siteconfig operator")
```

Reply:
How is it different?

Reply:
It must be deployed via siteconfig for this test to run

Reply:
IMO it makes sense given the context of the name of the test - `through siteconfig operator is successful in an IPv6 proxy-enabled environment with DHCP networking`.

i.e. what is being tested here is performing an image based installation through siteconfig operator.

 `Skip("Cluster must be deployed through siteconfig operator")` is just defining the constraint that the test must be run with the `ECO_LCA_IBI_SITECONFIG` value set to true. 

So I'm assuming you have the same feeling about `Skip("Cluster must use DHCP networking")`?

Reply:
yes (it's how someone looking at our tests from outside the team will perceive the message with an untrained eye)
And extra hard feelings about siteconfig operator - after realizing how confusing it is for those outside of the team.

Reply:
I think the clearest I can make this message is something like `Skip("ECO_LCA_IBI_SITECONFIG must be set to true")`. From my POV, I don't see how someone from the outside can glean any additional information from "Cluster was not opted to be deployed through siteconfig operator" vs "Cluster must be deployed through siteconfig operator"

Reply:
"Cluster must be deployed through siteconfig operator" seems like instruction to always follow to me